packages/service/tests/unit/test_discovery_client.py

```diff
@@ -319,6 +326,50 @@ def test___discovery_service_exe_unavailable___register_service___raises_file_no
         discovery_client.register_service(_TEST_SERVICE_INFO, _TEST_SERVICE_LOCATION)
 
 
+@pytest.mark.parametrize("programming_language", ["Python", "LabVIEW"])
+def test___registered_measurements___enumerate_services___returns_list_of_measurements(
+    discovery_client: DiscoveryClient, discovery_service_stub: Mock, programming_language: str
+):
+    grpc_service_descriptor = GrpcServiceDescriptor(
```

Comment:
Can we consider renaming this to something as `expected_service` or `expected_available_service` or `enumerate_services_response`?